vaadin-spring-tests/test-ts-services/frontend/src/test-component.js

```diff
@@ -2,13 +2,15 @@ import { PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { html } from '@polymer/polymer/lib/utils/html-tag.js';
 import * as appEndpoint from '../generated/AppEndpoint';
 import * as packagePrivateEndpoint from '../generated/PackagePrivateEndpoint';
+// import {AppEndpoint} from "../generated/AppEndpoint";
 
 class TestComponent extends PolymerElement {
   static get template() {
     return html`
         <button id="button">vaadin hello</button><br/>
         <button id="hello" on-click="hello">endpoint hello</button><br/>
         <button id="helloAnonymous" on-click="helloAnonymous">endpoint helloAnonymous</button><br/>
+        <button id="helloAnonymousWrapper" on-click="helloAnonymousWrapper">endpoint AppEndpoint.helloAnonymous</button><br/>
```

Comment:
Forgot to remove the `endpoint `?

Reply:
The button name should not matter, but forgot to remove the import comment. The test was failing to compile.

Fixed.